recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "7.4.2" %}
+{% set version = "7.6.4" %}
```

Comment:
This is wrong b/c the URL is hardcoded to https://github.com/ivmai/bdwgc/archive/gc7_4_2.tar.gz

Reply:
So true...mixed it up with another recipe...will fix it in a minute...

Reply:
Would it be possible to not hard code the URL?
Also the SHA1 was not updated (we'll support SHA1 very soon)

Reply:
Yes, fixing this in #5